src/shared/components/ResourceEditor/index.tsx

```diff
@@ -90,6 +94,14 @@ const ResourceEditor: React.FunctionComponent<ResourceEditorProps> = props => {
               onChange={onFormatChange}
             />
           )}
+          {!expanded && !isEditing && valid && (
+            <Switch
```

Comment:
I would put this switch before the expanded toggle, so the expanded toggle doesn't get pushed around if this one isn't moved. 

Also metadata is used as one word in or documentation, so we should do the same here. We can also be more consistent with our copy, using lowercase on all the button toggles here.

Reply:
Also, there should be a space between the toggles, like so `{" " }`

Reply:
Updated the code.